pkg/platform/provider/baremetal/phases/csioperator/images/images.go

```diff
@@ -28,11 +28,28 @@ import (
 
 const (
 	// LatestVersion is latest version of addon.
+	// TODO: bump up to v1.0.3
 	LatestVersion = "v1.0.2"
+
+	// Provisioner and ... csi sidecar and controller images name
+	Provisioner      = "csi-provisioner"
+	Attacher         = "csi-attacher"
+	Snapshotter      = "csi-snapshotter"
+	CSINodeRegistrar = "csi-node-driver-registrar"
+	TencentCBSDriver = "csi-tencentcloud-cbs"
 )
 
 type Components struct {
 	CSIOperator containerregistry.Image
+
+	// csi sidecar images
+	ProvisionerV101   containerregistry.Image
+	AttacherV110      containerregistry.Image
+	SnapshotterV110   containerregistry.Image
+	NodeRegistrarV110 containerregistry.Image
+
```

Comment:
don't add version info when define variable,  if you do this,  how to csi-operator upgrade case？

Reply:
This image is bind with various of backend storage in CSI-Operator v1.0.2. And this images is under csi-operator map.

In supporting upgrade CSI-Operator case， we need to add a new csi-operator  map in code to upgrade csi-operator other than backend storage system image.